src/components/Drawer/Drawer.tsx

```diff
@@ -200,6 +200,8 @@ class Drawer extends React.PureComponent<Props, never> {
 
     if (active && (mode === 'push' || mode === 'reveal')) {
       this.setBodyStyle();
+    } else {
+      document.body.className = '';
```

Comment:
@Farhan-emgage you can not do this. This is the exact issue you were encountering in messaging, where another component was removing the classes from the body element required by the drawer. It's not appropriate to remove all class names, and there may be another component also reliant upon a class on the body element.